### PR TITLE
Configure authentication for strict mode.

### PR DESCRIPTION
**Pull request by jeschkies**
Summary:
This configures Jenkins to authenticate as a DC/OS service account. So far the credentials
are passed as clear text in the config. We should follow up and use the credentials store
from Jenkins.

Serialization is still broken until #53 lands.

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/config.jelly [hunk 1]
```diff
@@ -22,6 +22,30 @@
...
 
+    <!-- TODO: Use credential selection instead. -->
+    <f:optionalBlock name="customSsl" title="Use a custom SSL certificate">
+      <f:entry title="SSL Certificate">
+        <f:textarea name="sslCert" value="${it.sslCert}"/>
+      </f:entry>
+    </f:optionalBlock>
+
+    <!-- TODO: Make the type selectable from None, DC/OS and Basic Authentication. -->
+    <f:optionalBlock name="authorization" title="Authorization">
```
Comment (jeschkies) [inline src/main/resources/org/jenkinsci/plugins/mesos/MesosCloud/config.jelly:33]: I basically want a drop list as described [here on Stackoferflow](https://stackoverflow.com/questions/39843036/jelly-config-expand-elements).

**Comment by jeschkies** [comment]
I think we can land this as is. It won't resolve [DCOS_OSS-5142](https://jira.mesosphere.com/browse/DCOS_OSS-5142) but at least provide a Docker image for testing. I update the ticket with action items.

**Review by kvish — APPROVED** [review]

File: src/main/resources/org/jenkinsci/plugins/mesos/MesosAgentSpecTemplate/config.jelly [hunk 2]
```diff
@@ -36,7 +36,7 @@
 
          <f:entry title="${%Mesos Offer Selection Attributes}" field="agentAttributes">
                  <f:textbox/>
-         </f:entry>git st
```
Comment (timcharper) [inline src/main/resources/org/jenkinsci/plugins/mesos/MesosAgentSpecTemplate/config.jelly]: lol

**Review by timcharper — APPROVED** [review]
Looks good!

**Pull request merged**